server/implementation/src/main/java/io/smallrye/graphql/websocket/graphqltransportws/GraphQLTransportWSSubprotocolHandler.java

```diff
@@ -28,7 +29,7 @@ public GraphQLTransportWSSubprotocolHandler(GraphQLWebSocketSession session, Map
 
     @Deprecated
     public GraphQLTransportWSSubprotocolHandler(GraphQLWebSocketSession session, ExecutionService executionService) {
-        this(session);
+        this(session, Collections.emptyMap());
```

Comment:
Should this be in this commit ? Is this not part of my prev commit ?

Reply:
Previously I blindly forward-ported https://github.com/smallrye/smallrye-graphql/commit/6e20902ba83b5c62013df61d6eb9711931b1a9e0 from 1.6.x, but didn't notice that it causes a compilation error because `main` includes https://github.com/smallrye/smallrye-graphql/commit/ab6484fe7f3b8f59dbd1a500bedb48275cbb0e5a which `1.6.x` doesn't

Maybe we should also add  https://github.com/smallrye/smallrye-graphql/commit/ab6484fe7f3b8f59dbd1a500bedb48275cbb0e5a into 1.6.x for consistency?